zerver/openapi/openapi.py

```diff
@@ -482,7 +482,15 @@ def validate_against_openapi_schema(
                     validator_value=brief_error_validator_value,
                     cause=error.cause,
                 )
-            message += f"\n\n{type(error).__name__}: {error}"
+            # Some endpoints have long, descriptive OpenAPI schemas
+            # which, when printed to the console, do not assist with
+            # debugging, so we only return the error.message.
+            if path in ["/register"]:
+                message += f"\n\n{type(error).__name__}: {error.message}"
+                message += "\nOpenAPI schema and validator schema omitted due to length of output. "
+                message += "See SchemaError message above for the endpoint that needs debugging."
```

Comment:
Would it make sense to include the URL of a section of the documentation in this message (or something we just print)?  I feel like that can really help, since I think most folks new to the codebase will have no idea __how__ to debug this.

We might need to write a new "Debugging schema validation errors" section that sets some context (we're verifying that every successful API response returned in the tests is a documented possibility in the API documentation), etc.